docker/account-lookup-service/default.json

```diff
@@ -0,0 +1,27 @@
+{
+  "ADMIN_PORT": 4001,
+  "API_PORT": 4002,
+  "DATABASE_URI": "mysql://account_lookup:password@mysql:3306/account_lookup",
```

Comment:
wouldnt this be 3007?

Reply:
I think you mean `3307`, and no, because `3307` is only exposed on the host machine. Internally to docker, the db is still on `3306`